R/bambu-extendAnnotations-utilityCombine.R

```diff
@@ -138,18 +138,22 @@ combineFeatureTibble <- function(combinedFeatureTibble,
                        pmax0NA(NSampleReadCount.new), 
                    NSampleReadProp = pmax0NA(NSampleReadProp.combined) + 
                        pmax0NA(NSampleReadProp.new), 
-                   NSampleGeneFDR = pmax0NA(NSampleGeneFDR.combined) + 
-                       pmax0NA(NSampleGeneFDR.new), 
-                   NSampleTxFDR = pmax0NA(NSampleTxFDR.combined) + 
-                       pmax0NA(NSampleTxFDR.new)) %>% 
+                   NSampleGeneScore = pmax0NA(NSampleGeneScore.combined) + 
+                       pmax0NA(NSampleGeneScore.new), 
+                   NSampleTxScore = pmax0NA(NSampleTxScore.combined) + 
+                       pmax0NA(NSampleTxScore.new),
+                    maxTxScore = pmax(maxTxScore.combined, 
+                        maxTxScore.new, na.rm = TRUE)) %>% 
             select(intronStarts, intronEnds, chr, strand, NSampleReadCount, 
-                   NSampleReadProp, NSampleGeneFDR, NSampleTxFDR, starts_with('start'),
-                   starts_with('end'), starts_with('readCount')) 
+                   NSampleReadProp, NSampleGeneScore, NSampleTxScore, maxTxScore, starts_with('start'),
+                   starts_with('end'), starts_with('readCount'), starts_with('equal')) 
```

Comment:
equal should be identical (if the same annotations are used), so you can select `equal=equal.combined` here (? might have to be tested)

Reply:
Resolved in bd92ed6acf6917a30d0969763cd7a61487ae5e1b